.travis.yml

```diff
@@ -6,9 +6,10 @@ notifications:
     on_success: change
     on_failure: always
     template: "%{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message} | %{build_url}"
-language: python
+# language: python
```

Comment:
I think so. They're sort of documenting the hack we had to do and which we would hope to take out if  and when travis supports python on Mac.